samples/test/samples.test.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC
+// Copyright 2022 Google LLC
```

Comment:
I think the prevailing wisdom right now is that we're supposed to leave the copyrights in place for modified files. You can also do `2020-2022`.